helper/testhelpers/postgresql/postgresqlhelper.go

```diff
@@ -12,44 +12,78 @@ import (
 )
 
 func PrepareTestContainer(t *testing.T, version string) (func(), string) {
-	return prepareTestContainer(t, version, "secret", "database")
+	env := []string{
+		"POSTGRES_PASSWORD=secret",
+		"POSTGRES_DB=database",
+	}
+
+	_, cleanup, url, _ := prepareTestContainer(t, "postgres", "postgres", version, "secret", "database", true, false, false, env)
+
+	return cleanup, url
 }
 
 func PrepareTestContainerWithPassword(t *testing.T, version, password string) (func(), string) {
-	return prepareTestContainer(t, version, password, "database")
+	env := []string{
+		"POSTGRES_PASSWORD=" + password,
+		"POSTGRES_DB=database",
+	}
+
+	_, cleanup, url, _ := prepareTestContainer(t, "postgres", "postgres", version, password, "database", true, false, false, env)
+
+	return cleanup, url
+}
+
+func PrepareTestContainerRepmgr(t *testing.T, name, version string, envVars []string) (*docker.Runner, func(), string, string) {
+	return prepareTestContainer(t, name, "bitnami/postgresql-repmgr", version, "secret", "", false, true, true, envVars)
+}
+
+func StopContainer(t *testing.T, ctx context.Context, runner *docker.Runner, containerID string) {
```

Comment:
Appears unused. Still want to keep this around?

Reply:
I actually intended to use this as part of the test rather than access the Docker runner stop/restart directly, so I'll switch over to that.